.gitlab-ci.yml

```diff
@@ -165,6 +203,16 @@ Integration:
         INTERNAL_NETWORK: ["true"]
         DISTRO_CODE: ["rhel_90"]
 
+Integration-nightly:
+  stage: test
+  extends: Integration
+  rules:
+    - if: '$CI_PIPELINE_SOURCE == "schedule" && $RUNNER == "aws/rhel-8.4-x86_64" && $SCRIPT != "libvirt.sh"'
+    # Run libvirt test on openstack because it has kvm support
+    - if: '$CI_PIPELINE_SOURCE == "schedule" && $RUNNER == "aws/rhel-8.4-x86_64" && $SCRIPT == "libvirt.sh"'
+      variables:
+        RUNNER: "openstack/rhel-8.4-x86_64"
+
 finish:
```

Comment:
Missing updates for the new API stage

Reply:
Added. I noticed a problem though, whenever someone adds a new test which will not be a part of tests package in the latest nightly, these tests will just fail. Current example is `api.sh aws.s3`

Reply:
That is a legitimate concern however it is off topic on the current PR. Open a ticket so we can deal with it appropriately.